relay-server/src/actors/processor.rs

```diff
@@ -569,13 +569,13 @@ impl EnvelopeProcessorService {
             config,
             envelope_manager,
             project_cache,
-            global_config,
```

Comment:
This isn't strictly required for this PR. I deleted it because it was causing trouble with types and it's currently not used anywhere, so it's very hard to predict how it'll be used at this point. We can reintroduce it when we use it.